client/src/pages/ErrorPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { Stack, Typography } from "@mui/material"
+import { Link, useRouteError } from "react-router-dom"
+
+export const ErrorPage = () => {
```

Comment:
I think it might be best to convert this to a not found page. Since other pages should handle any errors themselves.